integration_tests/tests/tasks/list.cy.ts

```diff
@@ -17,15 +17,72 @@ context('Tasks', () => {
     const allocatedTasks = taskFactory.buildList(5)
     const unallocatedTasks = taskFactory.buildList(5, { allocatedToStaffMember: undefined })
 
-    cy.task('stubReallocatableTasks', [...allocatedTasks, ...unallocatedTasks])
+    cy.task('stubReallocatableTasks', {
+      tasks: [...allocatedTasks],
+      allocatedFilter: 'allocated',
+      page: '1',
+      sortDirection: 'asc',
+    })
+
+    cy.task('stubReallocatableTasks', {
+      tasks: [...unallocatedTasks],
+      allocatedFilter: 'unallocated',
+      page: '1',
+      sortDirection: 'asc',
+    })
 
     // When I visit the tasks dashboard
     const listPage = ListPage.visit(allocatedTasks, unallocatedTasks)
 
     // Then I should see the tasks that are allocated
     listPage.shouldShowAllocatedTasks()
-
+    //
     // And the tasks that are unallocated
     listPage.shouldShowUnallocatedTasks()
   })
+
+  it('supports pagination', () => {
+    cy.task('stubAuthUser')
+
+    // Given I am logged in
+    cy.signIn()
+
+    const allocatedTasksPage1 = taskFactory.buildList(1)
+    const allocatedTasksPage2 = taskFactory.buildList(2)
+    const allocatedTasksPage9 = taskFactory.buildList(9)
+    const unallocatedTasks = taskFactory.buildList(1, { allocatedToStaffMember: undefined })
+
+    cy.task('stubReallocatableTasks', { tasks: [...allocatedTasksPage1], allocatedFilter: 'allocated', page: '1' })
```

Comment:
I don't *think* we need to spread these arrays into a new array, we can just do `tasks: allocatedTasksPage1`